components/date-picker/lib/popups/inner-popup.component.ts

```diff
@@ -59,8 +58,6 @@ export class InnerPopupComponent implements OnInit, OnChanges {
 
   constructor() {}
```

Comment:
Empty constructors could be removed, too?

Reply:
Ts will be generated constructor when has instance property, so no need to remove